_pages/monitoring-and-observability/insights.md

```diff
@@ -16,70 +16,63 @@ title: "Algorithmia Insights"
 
-Algorithmia Insights currently supports [Python 3.x](../clients/python#publishing-algorithmia-insights), [R](../clients/r#publishing-algorithmia-insights), and [Java](../clients/java#publishing-algorithmia-insights) algorithms and [Apache Kafka](https://kafka.apache.org/) as a destination for metrics.
+Algorithmia Insights is currently supported for [Python 3.x](/developers/algorithm-development/languages/python#publishing-algorithmia-insights), [R](/developers/algorithm-development/languages/r#publishing-algorithmia-insights), and [Java](/developers/algorithm-development/languages/java#publishing-algorithmia-insights) algorithms, and you can use [Apache Kafka](https://kafka.apache.org/) or [Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) as a broker for metrics.
 
 ### Types of Algorithmia Insights metrics
 
 Algorithmia Insights metrics fall into two categories:
 
-- Operational metrics
-- Inference-related metrics
-
-**Operational metrics** include (but are not limited to) execution time, request ID, algorithm owner, algorithm name, algorithm version, timestamp, etc. When Algorithmia Insights is turned on for a particular algorithm version, these are automatically collected and reported without any additional action necessary on the part of the algorithm author or manager.
-
-**Inference-related metrics** are defined by the algorithm author in algorithm code. Each metric is defined by a title, like `cute_cats_detected`, and a numeric value, like `24`. Multiple inference-related metrics can be reported at one time, and the set of metrics reported does not need to be the same from one algorithm execution to another.
+- **Operational metrics** include (but are not limited to) execution time, request ID, algorithm owner, algorithm name, algorithm version, timestamp, etc. When Algorithmia Insights is enabled for a particular algorithm version, these metrics are automatically collected and reported without any additional action necessary on behalf of the algorithm developer.
+- **Inference-related metrics** are defined in the algorithm source code itself. Each metric is defined by a title (e.g, `"cute_cats_detected"`) and the numeric value associated with that parameter in the code (e.g., the variable `cute_cats_detected`, which in this case hopefully evaluates to a large number like `24`). Multiple inference-related metrics can be reported at one time (see [Restrictions](#restrictions) below). Furthermore, because  metrics reporting is all defined within the code itself, the exact set of metrics reported does not need to be the same from one algorithm execution to another.
```

Comment:
```suggestion
- **Inference-related metrics** are defined in the algorithm source code itself. Each metric is defined by a title (e.g, `"cute_cats_detected"`) and the numeric value associated with that parameter in the code (e.g., the variable `cute_cats_detected`, which in this case hopefully evaluates to a large number like `24`). Multiple inference-related metrics can be reported at one time (see [Restrictions](#restrictions) below). Furthermore, because metrics reporting is all defined within the code itself, the exact set of metrics reported does not need to be the same from one algorithm execution to another.
```